alws/release_planner.py

```diff
@@ -608,10 +608,9 @@ async def commit_release(
                         build_ids=release.build_ids,
                     )
                     if issues:
-                        await move_issues(
+                        await close_issues(
                             github_client=github_client,
                             issues=issues,
-                            status=GitHubIssueStatus.RELEASED.value,
                         )
```

Comment:
we can move the logic with the github integration into a separate function and reuse it in the `alws/release_planner.py` and `alws/crud/errata.py` modules